docs/50-tutorials/artists.mdx

```diff
@@ -4,48 +4,52 @@
+If you put a [VRCFury | Toggle](/components/toggle) component in your prefab, it'll be added to the user's menu automatically at the menu path that you specify. You can toggle various things, including Objects, Blendshapes, or even arbitrary animation clips and other [actions](/actions)
 
-## Force an object to start as active, inactive, or deleted
+## Apply a property or delete an object when the avatar is uploaded
 
-Sometimes, users may do something which unexpectedly breaks your prop. For instance, they may "deactivate" the main object to temporarily hide it in the editor, while your animations only expect an inner child object to normally be deactivated. Using a VRCFury `Force Object State` feature, you can correct these issues for the user, forcefully activating, deactivating, or (in some rare cases) deleting any given object during their upload.
+Sometimes, users may do something which unexpectedly breaks your prop. For instance, they may "deactivate" the main object to temporarily hide it in the editor, while your animations only expect an inner child object to normally be deactivated. Using a [VRCFury | Apply During Upload](/components/other#apply-during-upload) component, you can correct these issues for the user, forcefully applying certain properties during upload.\
+In some rare cases, you might want to delete a given object during upload. This can be done using a [VRCFury | Delete During Upload](/components/other#delete-during-upload) component
 
 ## Editor gizmos
 
-Sometimes, you may want a user to drag a hidden object, such as a constraint target or particle emitter, into a specific spot on their avatar. Without an indicator, it may be difficult for users to align these objects properly. If you add a VRCFury component with a `Gizmo` feature to the object, you can create a custom editor gizmo on the object, to assist the user with custom text, an arrow, or wireframe sphere.
+Sometimes, you may want a user to drag a hidden object, such as a constraint target or particle emitter, into a specific spot on their avatar. Without an indicator, it may be difficult for users to align these objects properly. If you add a [VRCFury | Gizmo](/components/other#gizmo) component to the object, you can create a custom editor gizmo on the object, to assist the user with custom text, an arrow, or wireframe sphere.
 
 ## Test what VRCFury is going to do
 
-To check how vrcfury is going apply your changes to the avatar, simply go into unity's play mode. The changes will be applied to the play mode avatar object automatically and your changes should be visible.
+To check how VRCFury is going apply your changes to the avatar, simply go into unity's play mode and test using Gesture Manager. The changes will be applied to the play mode avatar object automatically and your changes should be visible.
+Alternatively, you can build a test copy by selecting the avatar's root and navigating to Tools > VRCFury > Build an Editor Test Copy

```

Comment:
I do not promote Editor Test Copies whatsoever anywhere. The more obscure the better.

Reply:
Gotcha